lambda_functions/v1/functions/lpa_codes/app/api/lets_see_about_this.py

```diff
@@ -7,9 +7,24 @@
 logger = custom_logger()
 
 
+def aws_credentials():
```

Comment:
Should this be in here? With the fake credentials?

Reply:
You're right. Should be using docker env vars instead. Good call :)